docs/contributing/code-review.md

```diff
@@ -0,0 +1,73 @@
+# Code review guidelines
+
+This document codifies our guidelines for doing code review
+and merging pull requests.
+
+> A Foolish Consistency is the Hobgoblin of Little Minds
+> - PEP 8
```

Comment:
So the idea of this main section is to add some general guidelines and then we go with specifics such as the terraform-related stuff, right?

Reply:
Yep! I'm hoping we can add other guidelines soon. At the least, we should have something for:

1. doc changes
2. New hub creations
3. Hub config changes
4. ???

Reply:
Those additional sections make sense to me. Not sure what else could be different from the general guidelines to deserve a specific section.